### PR TITLE
Fix local bootstrap for RHACS_TARGETED_OPERATOR_UPGRADES mode

### PR DESCRIPTION
**Pull request by kurlov**
## Description

- Install necessary pull secrets for operator namespace
- download necessary images

## Checklist (Definition of Done)

~~- [ ] Unit and integration tests added~~
- [x] Added test description under `Test manual`
~~- [ ] Documentation added if necessary (i.e. changes to dev setup, test execution, ...)~~
- [ ] CI and all relevant tests are passing
~~- [ ] Add the ticket number to the PR title if available, i.e. `ROX-12345: ...`~~
~~- [ ] Discussed security and business related topics privately. Will move any security and business related topics that arise to private communication channel.~~
~~- [ ] Add secret to app-interface Vault or Secrets Manager if necessary~~
~~- [ ] RDS changes were e2e tested [manually](../docs/development/howto-e2e-test-rds.md)~~
~~- [ ] Check AWS limits are reasonable for changes provisioning new resources~~

## Test manual

```
# Reset the cluster
colima k reset

# bootstrap and deploy using the script
scripts/install_operator_canary.sh
```

### CODE REVIEW

File: dev/env/scripts/create-imagepullsecrets [hunk 1]
```diff
@@ -70,7 +70,7 @@ function print_auth() {
 
 registry_auth="$(print_auth "$(mkauth "${username}" "${password}")")"
 
-if [[ "$INSTALL_OPERATOR" == "true" ]]; then
+if [[ "$INSTALL_OPERATOR" == "true" || "$RHACS_TARGETED_OPERATOR_UPGRADES" == "true" ]]; then
```
Comment (ludydoo) [inline dev/env/scripts/create-imagepullsecrets:73]: Can you explain why this change is required?

**Review by SimonBaeumer — APPROVED** [review]
